src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Login | Personal Dashboard</title>
+        <link rel="stylesheet" type="text/css" href="${pageContext.request.contextPath}/css/style.css" />
+    </head>
+    <body>
+        <nav id="main-menu">
+        </nav>
+        <section>
+            <div class="container centered">
+              <form action="${pageContext.request.contextPath}/login" method="post">  
+                <label>Userame:</label>
+                <input type="text" name="username"><br>  
```

Comment:
Never use the `<br>` tag to separate HTML elements.
`<br>` is a strictly text-related tag used to break line in a flowing text.
Use CSS to specifiy how HTML elements are layed out on the page.